app/lib/frontend/handlers/account.dart

```diff
@@ -31,6 +33,56 @@ final _logger = Logger('account_handler');
 /// Handles requests for /authorized
 shelf.Response authorizedHandler(_) => htmlResponse(renderAuthorizedPage());
 
+/// Handles GET /sign-in
+Future<shelf.Response> startSignInHandler(shelf.Request request) async {
+  if (!requestContext.experimentalFlags.useNewSignIn) {
+    return notFoundHandler(request);
+  }
+  final nonce = createUuid();
+  // TODO: get current session or create a new one
+  // TODO: store nonce on session
+  final oauth2Url = await authProvider.getOauthAuthenticationUrl(
+    state: '', // TODO: use meaningful state
```

Comment:
We need a uuid for state too, I think:
https://developers.google.com/identity/openid-connect/openid-connect#confirmxsrftoken

I think it's common to also include a URL to which we want to redirect back to. But we can take that later.